src/navigationDrawer/navigationDrawer.component.tsx

```diff
@@ -67,6 +73,21 @@ ForwardRefLink.displayName = 'ForwardRefLink';
 
 class NavigationDrawer extends Component<CombinedNavigationProps> {
   private createLink(plugin: PluginConfig, index: number): React.ReactElement {
+    const imgSrc =
+      this.props.darkMode && plugin.logoDarkMode
+        ? plugin.logoDarkMode
+        : !this.props.darkMode && plugin.logoLightMode
+        ? plugin.logoLightMode
+        : undefined;
+
+    const displayText = plugin.displayName
+      ? imgSrc
+        ? plugin.displayName
+        : plugin.logoAltText
+        ? plugin.logoAltText + plugin.displayName
+        : plugin.displayName
+      : plugin.plugin;
```

Comment:
This is quite confusing and hard to mentally process - I had to use this: https://converter.website-dev.eu/ to help me understand it! Maybe we should ignore prettier a little bit here and add some "unnecessary" brackets or indenting, or this could be refactored.

Reply:
Maybe this works a little better?

```
plugin.displayName
      ? imgSrc || !plugin.logoAltText
        ? plugin.displayName
        : plugin.logoAltText + plugin.displayName
      : plugin.plugin;
```

Reply:
I simplified the logic for setting `imgSrc` - we don't need to explicitly check whether `plugin.logoDark/LightMode` is thruthy or undefined since we'd be setting `imgSrc` to be undefined anyway (and in the event it's an empty string, the later checks on `imgSrc` will behave the same as if it were undefined).

Split the `displayText` assignment into a variable for the prefix (either `logoAltText` or `''`) and then `displayName` or `plugin` for the rest of it.

Hopefully by removing the nested ternary operators makes it easier to follow (it looks better to me at least).